src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,12 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+function TodoList(props) {
+    return (
+        <div className="todo-list">
+            {props.tasks.map(task => (<Todo key={task.id} task={task} />))}
```

Comment:
this looks great, really clean and I love the fact you're mapping through whatever is being passed as props, this is what makes react shine. Lets you spend less time building boilerplates and more time creating.